### PR TITLE
Soft fail command step docs

### PR DESCRIPTION
**Pull request by plasticine**
Adds docs over the `soft_fail` command step field ⚠️💖

### CODE REVIEW

**Comment by harrietgrace** [comment]
Looking great @plasticine! Don't hesitate to ping me if you've got any questions 😄

File: pages/pipelines/command_step.md.erb [hunk 1]
```diff
@@ -46,6 +46,17 @@ steps:
...
+  <tr>
+    <td><code>skip</code></td>
+    <td>
+      Whether to skip this step or not. Passing a string provides a reason for skipping this command. Passing an empty string is equivalent to <code>false</code>.
+      Note: Skipped steps will be hidden in the pipeline view by default, but can be made visible by toggling the 'Skipped jobs' icon.<br>
+      <em>Example:</em> <code>true</code><br>
+      <em>Example:</em> <code>false</code><br>
+      <em>Example:</em> <code>"My reason"</code>
+    </td>
+  </tr>
```
Comment (plasticine) [inline pages/pipelines/command_step.md.erb]: LOL — not sure how this ended up in the diff, whoops! 😅

File: pages/pipelines/command_step.md.erb [hunk 2]
```diff
@@ -137,6 +137,17 @@ _Optional attributes:_
       <em>Example:</em> <code>"My reason"</code>
     </td>
   </tr>
+  <tr>
+    <td><code>soft_fail</code></td>
+    <td>
+      Should this step cause the build to fail if it exits with a non-zero status.
```
Comment (harrietgrace) [inline pages/pipelines/command_step.md.erb]: Could we use the same format as we've got down in the retry section? I think the extra type info is handy 😄 
e.g. "A boolean value that defines whether or not this job can be retried manually."

Comment (plasticine) [reply pages/pipelines/command_step.md.erb]: Sure thing—I’ll make that change 👌🏼

Comment (plasticine) [reply pages/pipelines/command_step.md.erb]: Done! I split out the `exit_status` attr, but left the documentation of the actual `soft_fail` attr as it was (seemed more consistent).

**Comment by toolmantim** [comment]
Looks ace! I'll let @harrietgrace and you get this one over the line 🙌🏼

**Comment by plasticine** [comment]
Sweet! @harrietgrace and I had already chatted via pings so I think we’re good to go!

**Pull request merged**